var/spack/repos/builtin/packages/py-wheel/package.py

```diff
@@ -78,6 +78,7 @@ class PyWheel(Package, PythonExtension):
     depends_on("python +ctypes", type=("build", "run"))
     depends_on("python@3.7:", when="@0.38:", type=("build", "run"))
     depends_on("py-pip", type="build")
+    depends_on("python-venv", type=("build", "link", "run"))
```

Comment:
The way I think about it is `link` means both venv and python are for the target machine, whereas `build` means python could be for host while venv is for target. Run is similar to link, I guess, but also signals it's not needed during the build, whereas here it is.

It all doesn't matter too much since Python will never support cross compilation ;p

Alternative explanation: it's link because we symlink bin/python :laughing:

Reply:
But you don't use the compiler wrappers. Sounds like build/run to me.

Reply:
Sigh, I was looking at the wrong thing. My comment applies to `python-venv` depending on `python`.